Actividad1/Act2.5.4.html

```diff
@@ -119,7 +76,7 @@
             });
 
             $("#boton2").click(function(){
-                var radios = $("input:radio[name=radio]");
+                const radios = $("input:radio[name=radio]");
                 var str = "";
 
                 for(var i = 0; i < radios.length; i++){
```

Comment:
refactoriza este trocito con usando esto https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

Reply:
No puedo usar for each ya que $("input:radio[name=radio]") devuelve un jquery en lugar de un array por lo que tengo que usar el each de Jquery